packages/app/src/components/views/publication/CreatePostView.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, FormHelperText, Grid, TextField } from "@mui/material"
+import React from "react"
+import { Controller, useForm } from "react-hook-form"
+import { usePublicationContext } from "../../../services/publications/contexts"
+import { palette } from "../../../theme"
+import { ViewContainer } from "../../commons/ViewContainer"
+import PublicationPage from "../../layout/PublicationPage"
+import * as yup from "yup"
+import { yupResolver } from "@hookform/resolvers/yup"
+import { Article } from "../../../models/publication"
+import { useNavigate } from "react-router-dom"
+
+const articleSchema = yup.object().shape({
+  title: yup.string().required(),
+  article: yup.string().required(),
+})
+
+export const CreatePostView: React.FC = () => {
+  const navigate = useNavigate()
+  const { publication, draftArticle, saveDraftArticle } = usePublicationContext()
+
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(articleSchema),
+    defaultValues: draftArticle,
+  })
+
+  const onSubmitHandler = (data: Article) => {
+    saveDraftArticle(data)
+    navigate("/publication/preview-post")
+  }
+  
+
+  return (
+    <PublicationPage publication={publication} showCreatePost={false}>
+      <form onSubmit={handleSubmit((data) => onSubmitHandler(data as Article))}>
+        <ViewContainer maxWidth="sm">
+          <Grid container gap={4} flexDirection="column" mt={12.5}>
+            <Grid item xs={12}>
+              <Controller
+                control={control}
```

Comment:
```suggestion
                control={control}
                defaultValue=""
```

To prevent "A component is changing an uncontrolled input to be controlled." error.

Reply:
Aha, but it seems like `title` is still set to `undefined` somehow. Because I still get that error in the console if `defaultValue=""` is not added.

It's strange that it's only a problem with the `title` field and not the `article` field.